src/webgpu/shader/execution/expression/expression.ts

```diff
@@ -48,6 +60,9 @@ export type Config = {
   // If the number of test cases is not a multiple of the vector width, then the
   // last scalar value is repeated to fill the last vector value.
   vectorize?: number;
+  // The evaluation mode used when 'inputSource' is 'const'. If undefined, then an appropriate mode
+  // will be picked based on the input types.
```

Comment:
nit: These comments should be doc-comments.

Reply:
Done